proj1/src/test/java/com/codenvy/example/java/ApplianceTest.java

```diff
@@ -38,16 +77,100 @@ public void test3()
          assertEquals(State.ON,ob.getStatus());
     
 }
-
 @Test 
 public void test4()
 {
         Details ob=new Details("AC");
  
   ob.switchStatus();
   assertEquals(State.OFF,ob.getStatus());
- 
+  ob.switchStatus();
+  assertEquals(State.ON,ob.getStatus());
 }
 
+@Test
+public void test5()
+{
+    Appliance.size=0;
+    Appliance.perform(1,"AC",0);
+    Appliance.perform(1,"WH",0);
+    Appliance.perform(1,"CO",0);
+    assertEquals(3,Appliance.size);    
+}
+@Test 
+public void test6()
+{
+    
+    Appliance a=new Appliance();
+    Appliance.size=0;
+    Appliance.perform(1,"AC",0);
+   // Appliance.perform(1,"WH",0);
+    //Appliance.perform(1,"CO",0);
+    long startTime=System.currentTimeMillis();
+   
+   while(true)
+   {
+         float diff=(System.currentTimeMillis()-startTime);
+                       diff=diff/1000;
+                       float limit=1.1f;
+                       
+                       if(diff>=limit)
+                    {
+                        break;
+                    }
+                       
+   }
+                      Details ob=Appliance.objects.get(0);
```

Comment:
1. Indentation should be proper.
2. Give proper name to test functions.
3. Don't use 1, 2, 3 instead use enums.